docs/source/dev/py_postprocessing.rst

```diff
@@ -80,87 +66,58 @@ The module names should end on ``_widget.py``.
 To shorten the import statements for the widgets, please also add an entry in the ``__init__.py`` file of the ``jupyter_widget`` directory.
 
 There is a base class for visualization found in ``base_widget.py`` which already handles most of the widget logic.
-It also implements widgets for scan, simulation and iteration selection.
 
 .. autoclass:: picongpu.plugins.jupyter_widgets.base_widget.BaseWidget
    :members:
    :private-members:
 
-When created with an ``experiment_path`` argument, it assumes that there will be a number of subdirectories containing scan results named ``scan_<nr>``.
-Within each such directory it further assumes one or many PIConGPU simulation output directories named ``sim_<nr>``.
-The base class provides functionality to easily allow switching between visualizations for different scans or simulations.
+It allows to switch between visualizations for different simulation times (iterations) and different simulations.
```

Comment:
maybe replace visualizations -> widgets?

Reply:
Well we are not really switching between the widgets but only between plots for different iterations/times.
In the other repo there is also a class that allows switching between different visualizers. Would you want to have that as a contribution as well?